commands/root.go

```diff
@@ -70,7 +67,7 @@ var RootCmd = &cobra.Command{
 			return
 		}
 
-		if err := c.Exec(cmd.Context(), args...); err != nil {
+		if err := c.Exec(cmd.Context(), os.Args[1:]...); err != nil {
```

Comment:
Why is this not `args[1:]...` ?

Reply:
We need flag parsing (`DisableFlagParsing: false,`) to be able to work with persistent flags. This leads to `flag` parsing the flags and not adding them to `args`.

As a result, we need to get `os.Args[1:]` to exluce the binary name and pass the rest of the arguments to the Legacy CLI.

Example:

```console
# Using args
platform app:list -p XXX -e main
Enter a number to choose a project:
...

# Using os.Args[1:]
platform app:list -p XXX -e main
Applications on the project YYY (XXX), environment Main (main) (type: production):
+----------------+-------------+
| Name           | Type        |
+----------------+-------------+
...
```